src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -115,55 +51,140 @@ const ChatItemMyTextV2 = ({
   message = '',
   status = ChatStatus.PENDING,
   avatar,
-  chatType
+  chatType,
+  messageType,
+  data
 }: ChatItemMyTextProps) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good to see that you're using TypeScript for type safety. However, it seems like the `avatar` prop is not being used in this component. If it's not necessary, consider removing it to reduce complexity and improve maintainability.

```diff
-  avatar,
   chatType,
   messageType,
   data
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->